kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -48,6 +48,9 @@
     <failsafe.excluded.groups/>
     <!-- Sybase driver leaks its Timestamp implementation. To correctly handle it we need to add driver as additional test classpath. -->
     <maven.jdbc.driver.jar/>
+    <!-- No policy activated by default but overwritten at kie-server-integ-tests-policy -->
+    <org.kie.server.policy.activate/>
```

Comment:
Just a check - this will add empty string as a system property, right?

Reply:
Right, though empty will behave as null, because here won't get anything:
https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/policy/PolicyManager.java#L94-L95

Reply:
I see, thanks!